tripy/tripy/frontend/__init__.py

```diff
@@ -17,3 +17,6 @@
 
 from tripy.frontend.tensor import Tensor
 from tripy.frontend.trace import Trace
+from tripy.frontend.cache import ExecutableCache
+
+global_cache = ExecutableCache()
```

Comment:
IMO we should put this into `cache.py` rather than `__init__.py`.

Reply:
moved